dist/utils/webStorage.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+class WebStorageAdapter {
+    static stringifyValues(value) {
```

Comment:
This method may be usefull in more than an adpater, but in order to optimise my time, I'll place this method in a separate file when a 2nd adapater requires it.